aries-site/src/pages/templates/global-header.mdx

```diff
@@ -1,73 +1,127 @@
+The main navigation in the center of the header provide marketing content. The links allow users to find
+HPE products, contact support and get more information about HPE. These should also typically be enabled
+for most sites.
 
-On mobile, the Avatar should remain in the header, but the remaining nav items should be moved into the sidebar menu.
+![Global Header Main Navigation example](/templateImages/globalheader-nav.png)
 
-<Example
-  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/extend/global-header/HeaderNav.js"
-  horizontalLayout
-  height="none"
->
-  <HeaderNavItems />
-</Example>
+### Search
+
+The *Search* icon allows the user to search hpe.com. 
+
+<Search />
+
+### Shopping Cart
+
+The *Shopping Cart* icon goes to the HPE shopping cart for buy.hpe.com. This should only
+be enabled for sites which have products or solutions for sale.
+
+<Shop />
+
+### Country Selector
+
+The country selector allows the user to go to a language-specific version of hpe.com. This
+should be disabled for most sites since the link really only takes you back to hpe.com
+instead of staying on your site.
+
+<Language />
+
+### Sign in
+
+Sign in allows the user to sign in to their HPE Passport account for hpe.com. Enable this
+option if your site uses features that require a Passport login.
+
+![Sign In button](/templateImages/globalheader-sign-in.png)
+
+### Cube Menu
+
+The *Cube Menu* allows access to HPE services available to the user such as GreenLake and
+other Cloud console applications as well as general HPE links for HPE content like
+developer.hpe.com, education.hpe.com, etc. This menu should be enabled to allow access
+to other HPE sites and applications.
+
+<AppsRounded />
+
+## What makes up the Global footer
+
+The global footer contains links for privacy information and various opt-out and choices required by
+European GDPR standards as well US and other privacy requirements. In general these links should
+be enabled for customer-facing sites.
+
+### Privacy Link
+
+The *Privacy* link goes to HPE's Privacy Statement which explains how HPE manages and protects a users
+information and respects the relevant privacy standards like GDPR, APEC, etc.
+
+### Terms of Use Link
+
+The *Terms of Use* link goes to the legal statement that governs the user's activity on HPE's websites.
+
+### Ad Choices and Cookies Link
+
+The *Ad Choices and Cookies* link goes to the section of the *Privacy Statement* that explains how
+cookies are used as well as how you can opt-out of targetted advertisements and analytics data
+such as Google Analytics and Adobe Analytics.
+
+
+### Do Not Sell My Personal Information Link
+
+The *Do Not Sell My Personal Information* link goes to information on how to opt-out of any sale
+of personal collected by automatic data collection tools.
```

Comment:
Maybe include links again for convenience? Soft opinion.

```suggestion
of personal collected by automatic data collection tools.


<Box direction="row" gap="medium">
    <Button label="View Implementation Details" href="https://h50007.www5.hpe.com/hfws-test-pages/common-hfws-test-page.php" target="_blank" rel="noreferrer  noopener" secondary />
    <Button label="Development Team Contact" href="mailto:hpecom-cross-platform-development@hpe.com?subject=Common%20Header-Footer%20Service" secondary />
</Box>
```

Reply:
done